crates/vhost/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [Unreleased]
 
 ### Added
+- [[#165]](https://github.com/rust-vmm/vhost/pull/165) vhost: vdpa: Provide custom set_vring_addr() implementation
```

Comment:
IMHO this is more a fix then a new feature.
I would move it to the Fixed section.